### PR TITLE
Generating SoftPub bindings

### PR DESCRIPTION
**Pull request by nathan-alden-sr**

### CODE REVIEW

File: sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct DRIVER_VER_INFO
+    {
+        [NativeTypeName("DWORD")]
+        public uint cbStruct;
+
+        [NativeTypeName("ULONG_PTR")]
+        public IntPtr dwReserved1;
```
Comment (tannergooding) [inline sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs:16]: `ULONG_PTR` should be `UIntPtr`

File: sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct WTD_GENERIC_CHAIN_POLICY_CREATE_INFO
+    {
+        [NativeTypeName("_WTD_GENERIC_CHAIN_POLICY_CREATE_INFO::(anonymous union at C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/um/SoftPub.h:157:5)")]
+        public _Anonymous_e__Union Anonymous;
+
+        [NativeTypeName("HCERTCHAINENGINE")]
+        public void* hChainEngine;
```
Comment (tannergooding) [inline sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs:16]: Handle should be `IntPtr`?

File: sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs (hunk 1 above)
Comment (nathan-alden-hp) [reply sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs]: Fixed.

File: sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs (hunk 2 above)
Comment (nathan-alden-hp) [reply sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs]: Fixed.

File: sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs (hunk 1 above)
Comment (nathan-alden-sr) [reply sources/Interop/WinTrust/um/SoftPub/DRIVER_VER_INFO.cs]: Fixed.

File: sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs (hunk 2 above)
Comment (nathan-alden-sr) [reply sources/Interop/WinTrust/um/SoftPub/WTD_GENERIC_CHAIN_POLICY_CREATE_INFO.cs]: Fixed.

**Pull request merged**